doc/howtos/howto-3.html

```diff
@@ -28,7 +28,8 @@
 
 <p>
 Simply adding boomerang to a page and calling the <code>init()</code> method is sufficient to start the bandwidth
-test and beacon its results back to the server.  This is the code you'd use:
+test. If you want the beacon results to include the results of the bandwidth test, setting <code>block_beacon</code>
+to <code>true</code> will force boomerang to wait for the test to complete before sending the beacon.
```

Comment:
Can you add one line here stating that if you do not turn this on, you will only receive bandwidth results if they were cached in a cookie by a previous test run.